crates/astria-cli/src/sequencer/sudo/fee_change.rs

```diff
@@ -0,0 +1,284 @@
+    /// Change Sequence Byte Cost Multiplier
+    SequenceByteCostMul(SequenceByteCostMul),
+    /// Change Bridge Lock Byte Cost Multiplier
+    BridgeLockByteCostMul(BridgeLockByteCostMul),
+    /// Change Bridge Sudo Change Base Fee
+    BridgeSudoChangeBaseFee(BridgeSudoChange),
+    /// Change ICS20 Withdrawal Base Fee
+    Ics20WithdrawalBaseFee(Ics20Withdrawal),
+}
+
+#[derive(Clone, Debug, clap::Args)]
+struct Transfer {
+    #[command(flatten)]
+    inner: ArgsInner,
+}
+
+impl Transfer {
+    async fn run(self) -> eyre::Result<()> {
+        let args = self.inner;
+        let res = submit_transaction(
+            args.sequencer_url.as_str(),
+            args.sequencer_chain_id.clone(),
+            &args.prefix,
+            args.private_key.as_str(),
+            Action::FeeChange(FeeChangeAction {
+                fee_change: FeeChange::TransferBaseFee,
+                new_value: args.fee,
+            }),
+        )
+        .await
+        .wrap_err("failed to submit FeeChangeAction::TransferBaseFee transaction")?;
+
+        println!("FeeChangeAction completed!");
+        println!("Included in block: {}", res.height);
+        Ok(())
+    }
+}
+
+#[derive(Clone, Debug, clap::Args)]
+struct BridgeInit {
+    #[command(flatten)]
+    inner: ArgsInner,
+}
+
+impl BridgeInit {
+    async fn run(self) -> eyre::Result<()> {
+        let args = self.inner;
+        let res = submit_transaction(
+            args.sequencer_url.as_str(),
+            args.sequencer_chain_id.clone(),
+            &args.prefix,
+            args.private_key.as_str(),
+            Action::FeeChange(FeeChangeAction {
+                fee_change: FeeChange::InitBridgeAccountBaseFee,
+                new_value: args.fee,
+            }),
+        )
+        .await
+        .wrap_err("failed to submit FeeChangeAction::InitBridgeAccountBaseFee transaction")?;
+
+        println!("FeeChangeAction completed!");
+        println!("Included in block: {}", res.height);
+        Ok(())
+    }
+}
+
+#[derive(Clone, Debug, clap::Args)]
+struct SequenceBaseFee {
+    #[command(flatten)]
+    inner: ArgsInner,
+}
+
+impl SequenceBaseFee {
+    async fn run(self) -> eyre::Result<()> {
+        let args = self.inner;
+        let res = submit_transaction(
+            args.sequencer_url.as_str(),
+            args.sequencer_chain_id.clone(),
+            &args.prefix,
+            args.private_key.as_str(),
+            Action::FeeChange(FeeChangeAction {
+                fee_change: FeeChange::SequenceBaseFee,
+                new_value: args.fee,
+            }),
+        )
+        .await
+        .wrap_err("failed to submit FeeChangeAction::SequenceBaseFee transaction")?;
+
+        println!("FeeChangeAction completed!");
+        println!("Included in block: {}", res.height);
+        Ok(())
+    }
+}
+
+#[derive(Clone, Debug, clap::Args)]
+struct SequenceByteCostMul {
+    #[command(flatten)]
+    inner: ArgsInner,
+}
+
+impl SequenceByteCostMul {
```

Comment:
I think it would be better to type out the whole name for this type. `SequenceByteCostMultiplier`